frontend/app/components/work-packages/wp-display-attr/wp-display-attr.directive.ts

```diff
@@ -110,20 +112,18 @@ export class WorkPackageDisplayAttributeController {
   }
 }
 
-function wpDisplayAttrDirective() {
+function wpDisplayAttrDirective(wpCacheService:WorkPackageCacheService) {
 
   function wpTdLink(scope,
                     element,
                     attr,
                     controllers) {
 
-    // Listen for changes to the work package on the form ctrl
-    var formCtrl = controllers[0];
```

Comment:
If this is removed, you should remove the `require` as well. Actually, us being able to remove the dependency of the form is something I quite like about this approach.